packages/ui/components/DataFeedDetails.vue

```diff
@@ -74,6 +72,7 @@ export default {
       prefetch: true,
       query: feed,
       variables() {
+        console.log(this.feedFullName)
```

Comment:
Is this log needed? If so, maybe it could use some context in addition to the printed variable.